tests/acceptance/PatternsCept.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$I = new AcceptanceTester($scenario);
+
+$I->wantTo('test patterns');
+
+$I->loginAsAdminCached();
+
+// Create a new page
+$I->wantTo('add new page');
+$I->seeElement('#menu-pages.menu-icon-page');
```

Comment:
Test works fine locally, but fails in CI. Maybe add a `waitForElement` here to wait for the admin page to load? 🤔